.github/workflows/deploy.yml

```diff
@@ -95,11 +91,9 @@ jobs:
         7z e -y "$CARGO_C_FILE.zip" -o"${env:USERPROFILE}\.cargo\bin"
 
     - name: Install stable
-      uses: dtolnay/rust-toolchain@stable
+      uses: dtolnay/rust-toolchain@master
```

Comment:
Q, is it reliable to rely on the master branch?

Reply:
See https://github.com/dtolnay/rust-toolchain#inputs

>  When passing an explicit toolchain as an input instead of `@rev`, you'll want to use "dtolnay/rust-toolchain@master" as the revision of the action.

Reply:
Great thank you